README.md

```diff
@@ -2,25 +2,29 @@
+code excerpts and examples that our data users will find helpful, such as Jupyter notebooks and shell scripts. The material in the repository is that used by the blog's authors to prepare the blog at the time it was written. In contrast to data tutorials, with blogs we make no effort to review and keep up to date Jupyter notebooks, shell scripts, or any other code-like material. Also, we do not provide the environment and software versions that we may have used in preparing blog material. In many cases, the python environement described below may work if you want to run a Jupyter notebook used to generate material for a blog.
+
 ## Summary of Tutorials
 
 | Data access method | Description   |
 | ------------- | ------------- |
-| [AWS Open Data](./AWS_Open_Data) | A file README.md describing GeoNet's data available through AWS Open Data |
-| [FDSN](./FDSN) | Demonstrates how to access data through GeoNet's different FDSN web services (Dataselect, Station and Event). These tutorials are applicable to **seismic**, **acoustic-infrasound**, and **tsunami gauge (full sample rate)** data sets. |
-| [FITS](./FITS) | Shows how to retrieve and use data from [FITS](https://fits.geonet.org.nz/api-docs/). FITS is used to access **daily GNSS position data**, **manually collected volcano data**, and **volcano data logger data (limited cases)**. FITS is in the process of being replaced by [Tilde](https://tilde.geonet.org.nz/), and will later be deprecated. |
-| [Tilde](./Tilde) | Shows how to retrieve data from GeoNet's [Tilde API](https://tilde.geonet.org.nz/v3/api-docs/). These turorails apply to **DART**, **envirosensor**, and **tsunami gauge (down-sampled)** data. Tutorials cover Tilde's data, stats, and data summary APIs.
+| [AWS Open Data](https://github.com/GeoNet/data-tutorials/tree/main/AWS_Open_Data) | A file README.md describing GeoNet's data available through AWS Open Data |
+| [FDSN](https://github.com/GeoNet/data-tutorials/tree/main/FDSN) | Demonstrates how to access data through GeoNet's different FDSN web services (Dataselect, Station and Event). These tutorials are applicable to **seismic**, **acoustic-infrasound**, and **tsunami gauge (full sample rate)** data sets. |
+| [FITS](https://github.com/GeoNet/data-tutorials/tree/main/FITS) | Shows how to retrieve and use data from [FITS](https://fits.geonet.org.nz/api-docs/). FITS is used to access **daily GNSS position data**, **manually collected volcano data**, and **volcano data logger data (limited cases)**. FITS is in the process of being replaced by [Tilde](https://tilde.geonet.org.nz/), and will later be deprecated. |
+| [Tilde](https://github.com/GeoNet/data-tutorials/tree/main/Tilde) | Shows how to retrieve data from GeoNet's [Tilde API](https://tilde.geonet.org.nz/v3/api-docs/). These turorails apply to **DART**, **envirosensor**, and **tsunami gauge (down-sampled)** data. Tutorials cover Tilde's data, stats, and data summary APIs.
```

Comment:
the links here does not need changing, the `(./` at the beginning will do the magic